src/main/webapp/404.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<link rel="stylesheet" href="404_style.css">
+	</head>
+	<body>
+		<div id="main">
+			<h1>Error 404</h1>
+			<p>Either you messed up or we did. Let's not play the blame game ; )</p>
+			<a href="/"><input type="button" id="button404" value="Go Back Home" /></a>
```

Comment:
Personally I would recommend using `<button onclick="window.location.href='/index.jsp';" id="button404">Go Back Home</button>`

Reply:
Not familiar with jsp. Why would this be a better approach?

Reply:
Ah it's actually not jsp, it's just plain html. It's just a button with a onclick function that redirects you to another url, which has the same functionality as your line 10. So there is no need to change.